new-pipeline/src/dashboards.js

```diff
@@ -669,10 +669,10 @@ function getHumanReadableOptions(filterName, options) {
           (version <= latestNightlyVersion - 1 ? goodOptions : badOptions)
```

Comment:
Looks like Aurora is actually sticking around, which makes our life easier: https://bugzilla.mozilla.org/show_bug.cgi?id=1362137#c1